plugins/reporters/spdx/src/main/kotlin/Extensions.kt

```diff
@@ -204,6 +204,7 @@ internal fun Package.toSpdxPackage(
             SpdxConstants.NONE
         } else {
             packageLicenseExpressions.reduce(SpdxExpression::and)
+                .simplify()
```

Comment:
This will now throw an exception if the input contains only two identical licenses.